app/src/main/java/org/simple/clinic/patient/PatientSearchHelpers.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.patient
+
+/**
+ * [Regex] for stripping patient names and search queries of white spaces and punctuation
+ *
+ * Currently matches the following characters
+ * - Any whitespace
+ * - Comma, Hyphen, SemiColon, Colon, Underscore, Apostrophe, Period
+ * */
+private val spacePunctuationRegex = Regex("[\\s;_\\-:,'\\\\.]")
+
+fun convertNameToSearchableForm(string: String) = string.replace(spacePunctuationRegex, "")
```

Comment:
What do you think of renaming this function to just `nameToSearchableForm()`? Similar to avoiding function names starting with `get`, `set`, `create`, etc., I've started avoiding verb prefixes.